pii_team_t1.py

```diff
@@ -28,10 +28,10 @@ def find_us_ssn(text) -> list:
 def find_email(text) -> list:
     """Finds all occurrences of an email address in a text string"""
     # match an email address
-    return []
+    return re.findall(r'^[a-zA-Z0-9+_.-]+@[a-zA-Z0-9.-]+', text)
```

Comment:
The ^ causes your regular expression to match only at the beginning of the string.  Sine your test case has the email somewhere in the middle of the string, it will return an empty list and give you and index out-of-range error.  You should remove the ^ - you can test this out on [regex101.com](https://regex101.com)

Reply:
Ok, thank you. I forgot to update my regular expression when I changed my test case.